horizontalintegration/xmc-hztl-foundation-2024-q1
# Feature FIS-405 Breadcrumb Cleanup

## What does this PR do and why?

This PR includes some code cleanup for the Breadcrumb component and resolves some UX issues at the smaller breakpoints.

## Screenshots or screen recordings

| Before | After |
| ------ | ----- |
| ![localhost_6006_iframe html_globals=viewport%3Amobile1 args= id=components-authorable-shared-site-structure-breadcrumb--breadcrumb viewMode=story (2)](https://github.com/user-attachments/assets/364d108a-3422-4717-b9a8-90634c0b94f2) | ![localhost_6006_iframe html_globals=viewport%3Amobile1 args= id=components-authorable-shared-site-structure-breadcrumb--breadcrumb viewMode=story](https://github.com/user-attachments/assets/604e307c-180c-4867-a6b5-e66511eda4f1) |

## How to set up and validate locally

1. Check out the branch _feature/FIS-405-breadcrumb-cleanup_.
2. From a command prompt, navigate to _./src/hztl-foundation_ and execute `npm run storybook`.
3. From Storybook, navigate to _Components/Authorable/Shared/Site Structure/Breadcrumb_ and confirm that the breadcrumb functions correctly across breakpoints.

## PR acceptance checklist

This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
